frontend/src/Components/LoginForm/LoginForm.jsx

```diff
@@ -30,6 +30,7 @@ const LoginForm = () => {
       if (response.ok) {
         const data = await response.json();
         updateUser(data.user);
+        await fetchUserData();
```

Comment:
if you don't need to wait on the data request at line 31, you can send out two requests at the same time with `Promise.all`